packages/components/src/components/ScalarToggle/ScalarToggle.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import { cva, cx } from '../../cva'
+import ScalarIcon from '../ScalarIcon/ScalarIcon.vue'
+
+const props = defineProps<{
+  modelValue: boolean
+  disabled?: boolean
+  id?: string
+}>()
+
+const emit = defineEmits<{
+  (e: 'update:modelValue', value: boolean): void
+}>()
+
+function toggle() {
+  if (!props.disabled) {
+    emit('update:modelValue', !props.modelValue)
+  }
+}
+
+const toggleClasses = cva({
+  base: 'relative h-5 w-8 cursor-pointer rounded-full bg-b-3 transition-colors duration-300',
+  variants: {
+    checked: { true: 'bg-c-accent' },
+    disabled: { true: 'cursor-not-allowed opacity-40' },
+  },
+})
+</script>
+<template>
+  <div
+    :aria-checked="modelValue"
```

Comment:
I think these aria attributes are supposed to be on the input, also it might be good to be able to set `aria-label` or `aria-labbelledBy` on the toggle